pac4j-oauth/src/main/java/org/pac4j/oauth/client/FacebookClient.java

```diff
@@ -81,6 +83,8 @@
     protected boolean requiresExtendedToken = false;
     
     protected StateApi20 api20;
+
+    protected String appsecretProof = null;
```

Comment:
Why do we need a new value? Cannot we use the `secret` and just have a boolean `useAppsecretProof` to say if we must send it?